demo/src/main/java/com/example/demo/dto/ResponseDTO.java

```diff
@@ -0,0 +1,33 @@
+package com.example.demo.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+@Getter
```

Comment:
If using lombok, why did we declare constructor explicitly ? Read more about the usage